### PR TITLE
{tools}[foss/2021a] Quandl v3.6.1 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/9c9ff800b7b0d2d6e23fd47d97ae0fb4 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/q/Quandl/Quandl-3.6.1-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('inflection', '0.5.1', {
+        'checksums': ['1a29730d366e996aaacffb2f1f1cb9593dc38e2ddd30c91250c6dde09ea9b417'],
+    }),
+    ('more-itertools', '8.8.0', {
+        'checksums': ['83f0308e05477c68f56ea3a888172c78ed5d5b3c282addb67508e7ba6c8f813a'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/q/Quandl/Quandl-3.6.1-foss-2021a.eb:25]:
```suggestion
```

Does this require `more-itertools` 8.8.0? Python 3.9.5 provides `more-itertools` 8.7.0.

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/21572e5cd03018a934c7abe1ffd3f80e for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/99a25d9811abced2e6d03e8355fd4e2c for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/0e5031cea7e898334fed7f81d0f99de7 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @Micket!

**Pull request merged**